sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java

```diff
@@ -87,7 +87,7 @@ private String getUserAgentFixedPart() {
         osName = osName.replaceAll("\\s", "");
         String geteUserAgentFixedPart =
             UserAgentContainer.AZSDK_USERAGENT_PREFIX +
```

Comment:
please also replace `UserAgentContainer.AZSDK_USERAGENT_PREFIX` with hard coded value for the test